SrfGen/srfinfo2vm.py

```diff
@@ -1136,7 +1139,7 @@ def load_args(logger: Logger = qclogging.get_basic_logger()):
         ),
     )
     parser.add_argument(
-        "-o", "--out-dir", help="directory to place outputs", default="autovm"
+        "-o", "--out-dir", help="directory to place outputs", default="VMs"
```

Comment:
Should this be
`Data/VMs`?

Reply:
Srfinfo2vm generally has been run from the data folder. Where does SRFgen / realisation generation run from?

Reply:
The root directory. That way all paths are relative to the Cybershake Root

Reply:
I'm apprehensive to change the behaviour of a script. When we refactor this one we can change it's behaviour?